.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: ruby
 rvm:
-  - 2.1.5
\ No newline at end of file
+  - 2.1.6
+before_install:
+  - |
+    if [ "$LUMOS" == "true" ]; then
```

Comment:
This is a hard one...currently the tests are setup as essentially 'integration' tests with 1 side of the test mocked and responses returned from Tx or Github to check passing.  SO this approach requires binding  real credentials.  IMHO Instead we should have unit tests in play that can run quickly on PRs and such and a functional test suite to run separately.